README.md

```diff
@@ -382,10 +382,11 @@ This library has been tested against the following Browser type and versions:
 
 # Generate Documentation
 API documentation is generated when `npm install` is run. 
-[Online Docs](http://clover.github.io/remote-pay-cloud/1.3.1-rc1.2/) and
-[Online API class Docs](http://clover.github.io/remote-pay-cloud-api/1.3.1-rc1.1/)
+[Online Docs](http://clover.github.io/remote-pay-cloud/1.3.1/) and
+[Online API class Docs](http://clover.github.io/remote-pay-cloud-api/1.3.1/)
 
 # Release Notes
+## Version 1.3.1
 ## Version 1.3.1-rc1.2
```

Comment:
The 1.3.1 or the 1.3.1-rc1.2?  Probably not, it is meant to be a list of versions and changes in them.